src/pages/migrating/guide/design.mdx

```diff
@@ -42,11 +42,11 @@ accessibility, collaboration, and efficiency for users.
 
 ### Kit migration
```

Comment:
I think this section looks good. However, I wonder if we should cross-reference this part of the page as a link somewhere in the paragraph on the Designing > Get started > [Step 2: Access the tools](https://carbondesignsystem-git-fork-aubrey-142608-carbon-design-system.vercel.app/designing/get-started/#step-2:-access-the-tools) section? So people can find the table and know where to look to get access to the other tools in the meantime if needed. Let me know what you think about that.

Reply:
Cool! We discussed in design crit and will put a link under designing/get-started/#additional-design-assets